test/integration/conftest.py

```diff
@@ -60,12 +67,29 @@ def container_path(tmpdir_factory, container_url, container_name) -> str:
     return slc_path
 
 
-@pytest.fixture(scope="session")
-def saas_connection_params(saas_host, saas_pat, saas_account_id, operational_saas_database_id) -> dict[str, Any]:
-    connection_params = get_connection_params(
-        host=saas_host,
-        account_id=saas_account_id,
-        database_id=operational_saas_database_id,
-        pat=saas_pat,
-    )
-    yield connection_params
+@pytest.fixture(scope='session')
+def db_schema() -> str:
+    return TEST_SCHEMA
+
+
+@pytest.fixture(scope='session')
+def language_alias() -> str:
+    return TEST_LANGUAGE_ALIAS
+
+
+@pytest.fixture(scope='session')
+def deployer_factory(
+        backend_aware_database_params,
+        backend_aware_bucketfs_params,
+        db_schema,
+        language_alias):
+    @contextmanager
+    def create_deployer(create_test_schema: bool = False):
```

Comment:
The meaning of the boolean parameter is not very obvious reading the code of the tests below. Would it be better to use a `**argv` here, so that clients are forced to use `deployed_factor(create_schema=True)`?

Reply:
`**kwaargs` won't make it easier to understand the meaning of the parameter by looking at the method signature. I will change all calls to include the parameter name.